editor/debugger/editor_profiler.cpp

```diff
@@ -594,6 +594,36 @@ Vector<Vector<String>> EditorProfiler::get_data_as_csv() const {
 	return res;
 }
 
+void EditorProfiler::_sort_column(int column, int) {
+	TreeItem *root = variables->get_root();
+
+	// Sort per category
+	for (TreeItem *cat = root->get_first_child(); cat != nullptr; cat = cat->get_next()) {
+		Vector<Pair<String, TreeItem *>> vec;
+		for (TreeItem *it = cat->get_first_child(); it != nullptr; it = it->get_next()) {
+			String trimmed{ it->get_text(column).trim_suffix(" ms").trim_suffix("%") };
```

Comment:
Use standard `=` syntax for variable assignment:

```suggestion
			String trimmed = it->get_text(column).trim_suffix(" ms").trim_suffix("%");
```